app/Observers/BankAccountObserver.php

```diff
@@ -0,0 +1,107 @@
+<?php
+
+namespace App\Observers;
+
+use App\BankAccount;
+
+class BankAccountObserver
+{
+  /**
+   * Handle the bank account "created" event.
+   *
+   * @param  \App\BankAccount  $bankAccount
+   * @return void
+   */
+  public function created(BankAccount $bankAccount)
```

Comment:
```suggestion
  public function created(BankAccount $bank_account)
```
lets use underscores for variable names